packages/editor-ui/src/stores/credentials.store.ts

```diff
@@ -236,7 +236,7 @@ export const useCredentialsStore = defineStore(STORES.CREDENTIALS, {
 				};
 			}
 		},
-		enableOAuthCredential(credential: ICredentialsResponse): void {
+		enableOAuthCredential(_credential: ICredentialsResponse): void {
```

Comment:
Do we even need this? 🤔 Perhaps it was used before to create a store watcher for when this method was called but  right now there's only one reference to that function and it's just a call(which does nothing since this method is empty)